closed/make/mapfiles/libjncrypto/mapfile-vers

```diff
@@ -24,6 +24,10 @@
 SUNWprivate_1.1 {
 	global:
             handleErrors;
+            load_crypto_library;
+            unload_crypto_library;
+            find_crypto_symbol;
```

Comment:
I don't think any of the four symbols above need to be exported; please remove them.

Reply:
Removed